docs/api/effector/Effect.md

```diff
@@ -254,12 +253,14 @@ console.log(fx.use.getCurrent() === handlerB)
 
 ## Effect Properties
 
-:::note
-You are not supposed to [forward](./forward.md) to parts of effect (even though it consists of events and stores), since it's a complete entity on its own. This behavior will not be supported
-:::
+You are not supposed to [sample](./sample.md) to parts of effect (even though it consists of events and stores), since it's a complete entity on its own. This behavior will not be supported.
```

Comment:
мб тут to на target заменить?

Reply:
@YanLobat Можешь предложить suggestion? я не оч понимаю какой именно to поменять.

Reply:
Maybe smth like this:
```suggestion
You are not supposed to use parts of effect (like `.done` and `.pending`) as a target in [sample](./sample.md) or [forward](./forward.md) (even though they are events and stores), since effect is a complete entity on its own. This behavior will not be supported.
```